types/chart.js/index.d.ts

```diff
@@ -130,7 +130,7 @@ declare namespace Chart {
     }
 
     interface ChartData {
-        labels?: string[] | string[][];
+        labels?: (string | string[])[];
```

Comment:
No problem. I saw linting fail, but had some difficulties running it locally 😊 .